charts/service-deployment/values.yaml

```diff
@@ -112,6 +112,25 @@ hpa:
         averageUtilization: 75
   behavior: {}
 
+vpa:
+  # -- Whether to deploy VPA rules
+  enabled: false
+  # -- VPA resource specification (accepts any valid VPA spec fields)
+  spec: {}
+  # Example:
+  # spec:
+  #   resourcePolicy:
+  #     containerPolicies:
+  #     - containerName: '*'
+  #       maxAllowed:
+  #         cpu: 2
+  #         memory: 4Gi
+  #       minAllowed:
+  #         cpu: 100m
+  #         memory: 128Mi
+  #   updatePolicy:
+  #     updateMode: "Auto"
```

Comment:
https://github.com/kubernetes/autoscaler/issues/8424 "Auto" is now deprecated in favor of ["Recreate"](https://github.com/kubernetes/autoscaler/pull/8426/files)